sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -132,6 +132,18 @@ export async function drainAllMessages(receiver: Receiver<{}>): Promise<void> {
   await receiver.close();
 }
 
+/**
+ * Returns a TestClientType for either a Queue or a Subscription
+ * @param useSessions 
+ */
+export  function getRandomeReceiverTestClientType(useSessions: boolean): TestClientType {
```

Comment:
```suggestion
export  function getRandomReceiverTestClientType(useSessions: boolean): TestClientType {
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/9950/commits/08e64917500aaf5ce634bec7f133caec860f786f